warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -2401,6 +2402,10 @@ public static void configureTypeMappings(ShardQueryConfiguration config, Iterato
         }
     }
 
+    public static void addOption(IteratorSetting cfg, String option, boolean value) {
+        cfg.addOption(option, Boolean.toString(value));
+    }
+
     public static void addOption(IteratorSetting cfg, String option, String value, boolean allowBlankValue) {
```

Comment:
I don't see a change here to actually leverage `QueryOptions.addOption()` so that we only populate the option in the iterator setting if the value is different. Please add it in.